rmap-loader-deposit-disco/pom.xml

```diff
@@ -183,14 +182,6 @@
     </dependency>
 
     <dependency>
-      <groupId>org.postgresql</groupId>
```

Comment:
The general idea was to have popular database drivers in the jar file, so that all that's necessary to connect to a particular DB is a JDBC URI (i.e. configuration).  Otherwise, a driver jar would have to added to the classpath at runtime (i.e. instead of doing `java -jar whatever.jar`, you'd do `java -cp`, specify all jars, and specify the main class to run.

Either way is fine.  If you want to revert the change (i.e. adding it back in), then we can add a mariaDB jar as a separate issue.  The mysql connector was removed due to licensing, leaving only postgresql and sqlite.  MariaDB looks like a viable alternative in order to enable mysql/mariadb connectivity out of the box